### PR TITLE
fix: when 'post.careta_at' is null caused the page to crash

### PR DESCRIPTION
**Pull request by yuxino**
- when 'post.careta_at' is null caused page crash, [CC](https://youtu.be/AMFM5nmFyVY) _(•̀ω•́, 

- Fix react 16.9's [warning](https://github.com/axetroy/blog/compare/master...yuxino:master#diff-a53f67632d7ed873853eac2338c6cae3L123-L124) ,  •̀ω•́)_ [CC](https://github.com/facebook/react/issues/16382)

### CODE REVIEW

File: src/pages/post/index.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ import actions from "../../redux/actions";
 import "./post.css";
 
 class Post extends Component {
+  isUnmount = false;
```
[the remark this answers is not in the document]
Comment (yuxino) [reply src/pages/post/index.js]: Has been deleted

File: src/pages/post/index.js [hunk 2]
```diff
@@ -226,9 +226,12 @@ class Post extends Component {
                   <span>
                     <Icon type="calendar" style={{ marginRight: "0.5rem" }} />
                     发布于&nbsp;
-                    {formatDistanceToNow(new Date(post.created_at), {
-                      locale: chinese
-                    })}
+                    {formatDistanceToNow(
+                      new Date(post && post.created_at ? post.created_at : 0),
```
Comment (axetroy) [inline src/pages/post/index.js]: Thank you very much for your PR to fix the bugs for me.

But this line seem doesn't not need at all. ref: [deepscan](https://deepscan.io/dashboard/#view=project&tid=5773&pid=7588&bid=79785&subview=pull-request&prid=178232)

Comment (axetroy) [reply src/pages/post/index.js]: It should be 

```jsx
{ post.created_at ? formatDistanceToNow(post.created_at) : '' }
```

Comment (yuxino) [reply src/pages/post/index.js]: @axetroy Ok, it's has been removed _(•̀ω•́,

**Comment by axetroy** [comment]
LTGM! Thanks

**Pull request merged**